app/views/posts/_form.html.erb

```diff
@@ -65,7 +65,7 @@
     </div>
     <div>
       <span class="has-float-right has-font-size-caption js-character-count-post-title"
-            data-max="255" data-min="15" data-display-at="0.75">
+            data-max="255" data-min="15" data-display-at="0.75" style="display:none">
```

Comment:
Instead of using an inline `style` attribute, add `hide` to the `class` attribute (and same for the other one).

Reply:
`class= hide` Did you mean it?

Reply:
Yes, but since the `span` element already has a `class` attribute, just add `hide` to the end of the list.